net/FlatBuffers/ByteBuffer.cs

```diff
@@ -632,12 +662,9 @@ public byte Get(int index)
 #endif
 
 #if ENABLE_SPAN_T
-        public unsafe string GetStringUTF8(int startPos, int len)
+        public string GetStringUTF8(int startPos, int len)
         {
-            fixed (byte* buffer = &MemoryMarshal.GetReference(_buffer.ReadOnlySpan.Slice(startPos)))
-            {
-                return Encoding.UTF8.GetString(buffer, len);
-            }
+            return Encoding.UTF8.GetString(_buffer.Span.Slice(startPos, len));
```

Comment:
It seems like the doc for [GetString](https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.getstring?view=netcore-3.1) only has an overload for ReadOnlySpan, but [Span.Slice](https://docs.microsoft.com/en-us/dotnet/api/system.span-1.slice?view=netcore-3.1) returns a Span. 

I just want to make sure this is correct and that we are compiling with ENABLE_SPAN_T enabled in our unit tests.

Reply:
Span<T> can behave as a ReadOnlySpan<T> through an implicit type conversion.
https://docs.microsoft.com/en-us/dotnet/api/system.span-1.op_implicit?view=netcore-3.1

Reply:
I realize that my argument is wrong.

The System.Buffers.Binary provided here is not from .NetStandard 2.1, but from the one that came with the System.memory that you get from Nuget. In that case, I found that you can't cast implicitly.

That would mean that the project file you have now is missing a dependency.

You have two options in this case
1.NetStandard was changed from 2.0 to 2.1
2. increasing dependency.
I recommend 1.

I realized that I was going to write some test code again, but I don't have a test project for Core in the first place. I'll add it once I've decided on the above policy.

(I wasn't sure if I should put this content here, but I'll put it here for now. Please move it to here if you need it.)

Reply:
Generally we want to keep flatbuffers as widely available as possible, so going to .NetStandard 2.0 to 2.1 might preclude some folks who aren't using the newer one.

What do you mean for #2 about increasing dependency? What dependency would we have to add?

Reply:
I understand.

Regarding #2, it will need to make a change to add a dependency to System.Memory and get it from Nuget.

Unfortunately, the Span that comes with System.Memory was created for backwards compatibility and has no performance advantage whatsoever.

So, to complicate things a bit, I propose Plan 3, which is compatible with both .NetStandard 2.0 and 2.1.

This plan adds a dependency in the case of 2.0 and uses the traditional unsafe code.
in the case of 2.1, it does not add a dependency and uses the pull request code I created.

How about this plan?

Reply:
Yes, I am going to use this technique.

Okay. So, is it correct that the condition is to enable Span<T> only when NETSTANDARD2_1 and ENABLE_SPAN_T?

Reply:
May I begin work on the above policy?

Reply:
Yeah, feel free. I would make the check to be >= NETSTANDARD2_1, however that would work.

Reply:
@dbaileychess the corrections have been completed, please confirm.